src/utils/page.ts

```diff
@@ -1,24 +1,15 @@
 import { base64UrlEncode } from '../utils/b64'
 import { replaceEmailsWithHashes } from '../utils/email'
 
-/**
- * @return {string}
- */
-export function loadedDomain () {
+export function loadedDomain (): string {
   return (document.domain || (document.location && document.location.host)) || (window && window.location && window.location.host) || 'localhost'
 }
 
-/**
- * @return {string|undefined}
- */
-export function getReferrer (win = window) {
+export function getReferrer (win = window): string | undefined {
```

Comment:
Maybe missing a type annotation for `win`? Same below

Reply:
updated 👍